.github/workflows/release-vscode-extension.yml

```diff
@@ -0,0 +1,104 @@
+# This action is triggered when tags of the form releases/vscode-smart-contracts/*.*.*
+# are created. It will build cargo-concordium and vscode extension and upload it
+# as artifacts of the workflow run. If a release already exists with the given
+# tag then the extension will also be attached as assets to the release.
+#
+# The intended workflow is that a release is created and tagged, at which point
+# this action will be triggered and complete the build.
+name: 'Build and publish VSCode extension in a release'
+on:
+  # This release action always requires a tag to publish a release.
+  # This job will run when a releases/cargo-concordium/$VERSION
+  # tag is created.
+  push:
+    tags:
+      - releases/vscode-smart-contracts/*.*.*
+
+jobs:
+  build-cargo-concordium:
```

Comment:
**Discussion:** This is using cargo-concordium from the same commit rather than at some release tag. 

Maybe this is not an issue in practice, but since this is building cargo-concordium, the versioning could be a bit off if we don't release the extension right after releasing cargo-concordium.
Say that we release cargo-concordium, then fix a bug in cargo-concordium and then release the extension, this bug fix will be shipped with the extension.